### ISSUE TITLE
New content for Theme and Customisation group in Components

### ISSUE THREAD

**Issue by GCHQ-Developer-718** [mi6/ic-design-system#1663]
### What component is this related to?
See new component groups ticket

#### Theme
Same information as the current [Theme page](https://design.sis.gov.uk/components/theme/#customising-component-colours).

Add a link to colour contrast guidance to help users select appropriate colours (or create a ticket if this guidance has not been added to the site) and the "Toggle dark mode" pattern.

Update the monochrome guidance to explain why the monochrome prop exists. This could be in the form of use/don't use (tick and cross) picture examples with a component on a coloured background that only meets contrast requirements if monochrome is used, with one picture showing the monochrome variant and the other showing the component without monochrome set. Consider linking to relevant WCAG success criteria for contrast requirements.

 Additionally, add an example of using the monochrome variant of a component to the Code tab, with developer-specific guidance to help users know when they should be using the monochrome prop.

#### Customize ICDS
This is guidance-only (no code).

Explain that custom components can be styled to fit in with ICDS components.

Provide a set of cards with links to necessary information and a quick overview of what the linked information is for:

- Style tokens (Styles > Tokens)
- Icons and images (styles)
- Page and component layout (patterns) or just the main patterns page

Explain that some components have built-in customisation options (e.g. respond to changes in branding or have css props like --input-width).

Accessibility section: explain that ICDS components have been tested for accessibility, but users are responsible for ensuring that any customisation meets accessibility policy. Provide links to the policy and "Testing for accessibility" on the main Accessibility page. 

### Why do we need it?

Provides a central location for developers to access guidance on theming, branding, dark mode and styling custom components to look like ICDS components using our Style tokens and patterns.

### 🚨 Urgency (low, medium or high)

If applicable, tell us how urgent it is that this issue gets resolved, based on the impact it has on your team's work or project timeline.

### Related open issues

After #1662 (Tokens restructure)

**Comment by GCHQ-Developer-299** [comment]
A user has raised the issue that the following section on the Theme page:
`:root {
  --ic-brand-color-primary-r: 92;
  --ic-brand-color-primary-g: 9;
  --ic-brand-color-primary-b: 72;
}`
doesn't actually seem to set the global brand colour. Only using the theme component with brandColor prop. 

Might be worth updating the theme guidance as part of this ticket.
